src/go/k8s/pkg/resources/certificates.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package resources
+
+import (
+	"fmt"
+
+	"github.com/vectorizedio/redpanda/src/go/k8s/apis/redpanda/v1alpha1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const (
+	// operatorClientCert cert name - used by kubernetes operator to call KafkaAPI
+	operatorClientCert = "operator-client"
+	// userClientCert cert name - used by redpanda clients using KafkaAPI
+	userClientCert = "user-client"
+	// adminClientCert cert name - used by redpanda clients using KafkaAPI
+	adminClientCert = "admin-client"
+	// redpandaNodeCert cert name - node certificate
+	redpandaNodeCert = "redpanda"
+
+	// adminAPIClientCert is client certificate for admin api
+	adminAPIClientCert = "admin-api-client"
+	// adminAPINodeCert cert name - node certificate for Admin API
+	adminAPINodeCert = "admin-api-node"
+
+	// cert-manager has limit of 64 bytes on the common name of certificate
+	nameLimit       = 64
+	separatorLength = 1 // we use - as separator
+)
+
+// Certificates provides namespaced names of all TLS certificates used for redpanda listeners
+type Certificates struct {
```

Comment:
Isn't this struct more like `CertificateNames`, it doesn't hold any certificate resources, it generates namespace, names.